release/release.go

```diff
@@ -151,6 +151,38 @@ func CheckUpstreamRelease(ctx context.Context, client *github.Client, org, repo
 	return releases, nil
 }
 
+func KubernetesGoVersion(ctx context.Context, client *github.Client, version string) (string, error) {
+	const (
+		owner string = "kubernetes"
+		repo  string = "kubernetes"
+	)
+
+	file, _, _, err := client.Repositories.GetContents(ctx, owner, repo, ".go-version", &github.RepositoryContentGetOptions{
+		Ref: version,
+	})
+
+	if err != nil {
+		switch err := err.(type) {
+		case *github.ErrorResponse:
+			if err.Response.StatusCode == http.StatusNotFound {
+				logrus.Debugf("Failed to get .go-version file from given Kubernetes version")
+			}
+			return "", err
+		default:
+			return "", err
+		}
+	}
+
+	goVersion, err := file.GetContent()
+
```

Comment:
```suggestion
```

Reply:
Done

Reply:
This change doesn't appear to have been pushed up.